media/gpu/BUILD.gn

```diff
@@ -204,32 +205,32 @@ component("gpu") {
     }
   }
 
-  if (use_v4lplugin) {
-    sources += get_target_outputs(":libv4l2_generate_stubs")
-    deps += [ ":libv4l2_generate_stubs" ]
-  }
-
   if (use_v4l2_codec) {
-    deps += [
-      "//third_party/libyuv",
-      "//ui/ozone",
```

Comment:
we are removing //ui/ozone dependency here. is this intentional?

Reply:
Sorry, I'm not sure so I need to research.

Reply:
It is not intentional. It looks like it got accidentally removed during one of the rebases of the original patch.

Reply:
I will fix to retain //ui/ozone.